product-suite/dual-investment-pools-dip/README.md

```diff
@@ -4,13 +4,13 @@ description: Improved DOVs
 
 # Dual Investment Pools (DIP)
 
-Dual Finance has upgraded decentralized option vaults (DOVs), by creating Dual Investment Pools (DIPs) offering streaming executable yield strategies. Customize strategies based on your risk appetite and earn yield immediately.\
+Dual Finance has advanced decentralized option vaults by creating Dual Investment Pools (DIPs), offering real-time yield strategies for your crypto assets. A variety of option strike prices and expiries are available and the premium ($USDC) and yield (APY%) are streamed live, allowing for greater transparency and ease of comparison across different DIPs. Users can customize strategies based on their own risk appetite and objectives.\
```

Comment:
A variety of option strike prices and expiries are available and the premium ($USDC) and yield (APY%) are streamed live

Maybe "A variety of strikes and expirations are available with premiums paid out in stablecoins (USDC). Yields are presented as % APY for ease of comparison across different DIPs that way users can customize.."

Reply:
Tweaked as per your feedback!